compose-table/src/main/java/org/dhis2/composetable/ui/DataSetTableUi.kt

```diff
@@ -360,8 +360,8 @@ fun ItemValues(
                     cellValue = cellValue,
                     nonEditableCellLayer = {
                         nonEditableCellLayer(
-                            columnIndex = cellValue.column ?: -1,
-                            rowIndex = cellValue.row ?: -1,
+                            columnIndex = cellValue.column!!,
+                            rowIndex = cellValue.row!!,
```

Comment:
Using `!!` paradigm would lead to unexpected exceptions, is there any possibility to make `column` and `row` non null?.
If not, maybe we are using `CellValue` object for two different responsibilities.

Reply:
Perhaps is something to talk about in future designs. I have refactored several nullable to make them no nullable.

After doing some testing, under some circumstances it was crashing with this change, so I'll leave them as before and control de error. Still only happen sometimes during recomposition, not sure why yet.